kuryr_kubernetes/controller/drivers/namespace_security_groups.py

```diff
@@ -39,7 +39,7 @@
 
 cfg.CONF.register_opts(namespace_sg_driver_opts, "namespace_sg")
 
-DEFAULT_NAMESPACE = 'default'
+DEFAULT_NAMESPACE = ['default']
```

Comment:
If we plan to allow some other namespaces to be considered as default (allowing all traffic from other namespaces), shouldn't this be config option?